### ISSUE TITLE
Stuck in Ice Barrows

### ISSUE THREAD

**Issue by Explodncheez** [Wynncraft/Issues#6479]
The pressure plate before the "Run to the End of the Room" doesn't seem to be working, so as of now, I'm stuck in the previous room. I would like to avoid using /kill if at all possible.
I have tried worldhopping. On some worlds, literally nothing happens. On others, text pops up and the ice break sound is heard, but the wall does not break.

**Edit:** This door seems to work once and only once on each newly-restarted world. Since going the rest of the dungeon solo at my level is suicide, I'm still stuck.

**Comment by Trixter313** [comment]
Here is the link to the Wiki for the quest: https://wiki.wynncraft.com/index.php?title=Ice_Barrows

Consult that to make sure you did everything that you need to do.

Unfortunately if it is broken the only way out would be to use /kill.

**Comment by ColinRioux** [comment]
It has been more than 30 days without a response to this issue, or there have been no further reports of this issue, or it is a Minecraft mechanic and not related to Wynncraft. If your issue still persists, feel free to make a new issue trying to describe it with as much detail as possible.

**Comment by LA1S0N** [comment]
i have the same issue as well.
